benefits/eligibility/templates/eligibility/start__login_gov.html

```diff
@@ -0,0 +1,52 @@
+{% extends "eligibility/start.html" %}
+{% load i18n %}
+{% load core %}
+
+{% block page_title %}
+  {% translate "eligibility.pages.start.login_gov.title" %}&nbsp;|&nbsp;
+{% endblock page_title %}
+
+{% block headline %}
+  <div class="col-lg-10">
+    <h1>{% translate "eligibility.pages.start.login_gov.headline" %}</h1>
+  </div>
+{% endblock headline %}
+
+{% block media-list %}
+  {% translate "core.icons.idcardcheck" context "image alt text" as icon_alt %}
+  {% translate "eligibility.pages.start.login_gov.start_item.heading" as heading %}
+  {% translate "eligibility.pages.start.login_gov.start_item.details" as details %}
+  {# djlint: off #}
```

Comment:
These are all over the template files now. Just out of curiosity, what does `djLint` not like about the code? The use of `{% media_item ...}`?

Reply:
> what does djLint not like about the code

Yeah it mostly gets tripped up on any custom block tag, like `{% media_item %} ... {% endmedia_item %}`